lib/log.js

```diff
@@ -209,9 +215,9 @@ const CodingGameServiceLog = new Lang.Class({
     reversedCopyFileEvents: function() {
       // Go through the event log in reverse and yield copy-file
```

Comment:
You could fix the indentation of these comments too, like you did for the code just below them.